nano/node/transport/udp.cpp

```diff
@@ -61,7 +70,16 @@ std::function<void(boost::system::error_code const &, size_t)> nano::transport::
 
 std::string nano::transport::channel_udp::to_string () const
 {
-	return boost::str (boost::format ("UDP: %1%") % endpoint);
+	return boost::str (boost::format ("%1%") % endpoint);
+}
+
+nano::transport::channel_udp::~channel_udp ()
+{
+	std::lock_guard<std::mutex> lk (channel_mutex);
+	if (type == nano::transport::transport_type::tcp && socket != nullptr)
```

Comment:
Mixing TCP in to UDP is causing this cleanup complexity, this is automatically handled by channel_tcp.

Reply:
Updated